setuptools_scm/git.py

```diff
@@ -130,7 +130,8 @@ def _list_files_in_archive():
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
     tf = tarfile.open(fileobj=proc.stdout, mode='r|*')
     for name in tf.getnames():
-        print(name)
+        if not isdir(name):
```

Comment:
instead of checking for `isdir` here, i believe loop should be `for member in tf.getmembers()`

`member.type` should not equal `tarfile.DIRTYPE`

Reply:
Indeed, that looks better, I have done that.